bin/yaml/kotlin.yaml

```diff
@@ -33,3 +33,4 @@ compilers:
       - 1.9.0
       - 1.9.10
       - 1.9.20
+      - 2.0.0
```

Comment:
While testing the change locally, I spotted the following issue: all Kotlin/JVM compiler versions depend on `compilers/java 15.0.2`, but in fact, they various Java versions up to `21.0.0`: https://github.com/compiler-explorer/compiler-explorer/blob/main/etc/config/kotlin.amazon.properties

I guess everything works fine on `godbolt.org` as it has all Java versions installed, but when only selected compiler versions are installed, it could become a problem.

If it's indeed an issue, I'll fix it

Reply:
Should be ok; thanks for thinking though.

Reply:
Both installed perfectly.

Reply:
Thank you for merging!